pixi-multistyle-text.ts

```diff
@@ -2,11 +2,11 @@
 
 "use strict";
 
-interface ExtendedTextStyle extends PIXI.TextStyleOptions {
+export interface ExtendedTextStyle extends PIXI.TextStyleOptions {
 	valign?: "top" | "middle" | "bottom";
 }
 
-interface TextStyleSet {
+export interface TextStyleSet {
```

Comment:
Yes, it's required - you can't export the `MultiStyleText` class if it has properties or methods that use types that wouldn't be accessible externally.